### ISSUE TITLE
eliminate memory copies in setup using fk::views

### ISSUE THREAD

**Issue by grahamlopez** [project-asgard/asgard#48]
ASGarD
    asgardasgard
    Issues
    #48

Open
Opened 3 months ago by Graham Lopez
Close issue
New issue
eliminate memory copies in setup using fk::views

depends on #47 (closed)
Related issues
0

Graham Lopez @bgl added help_wanted label 3 months ago
Graham Lopez @bgl · 1 month ago
Owner

The basic needed functionality should be present now. This issue is up for grabs and high priority. If nobody takes it by Monday (15 April 2019), then I will take it or David will otherwise assign.
Green, David L
Green, David L @dg6 · 1 month ago
Owner

Excellent!

@elwasif My understanding of this is that you can now proceed with profiling the setup routines, and as you do that, identify and replace the memory copies with use of the fk::views. Would you give this a try, please?
Graham Lopez
Graham Lopez @bgl · 1 month ago
Owner

@elwasif just let me know if you want to discuss this at any point; we can meet and I can show you briefly how to use the new views capability
    Graham Lopez @bgl assigned to @elwasif 1 month ago

**Comment by grahamlopez** [comment]
I'm closing this, as we have been taking the approach for a few months now to do profile-driven optimization and use views as necessary according to that process.
